vignettes/MetaboCoreUtils.Rmd

```diff
@@ -126,6 +126,48 @@ countElements(frml)
 ```
 
 
+## Retetion time indexing
+
+Retention times are often not directly comparable between two LC-MS systems, even
+if nominally the same separation method is used. Conversion of retention times to
+retetion indices can overcome this issue. The `MetaboCoreUtils` package provides
+a function to perform this conversion. Below we use an example based on indexing
+with a homologoues series af N-Alkyl-pyridinium sulfonates (NAPS).
+
+```{r}
+rti <- read.table(system.file("retentionIndex/rti.txt",
+                              package = "MetaboCoreUtils"),
+                  header = TRUE,
+                  sep = "\t")
+  
+rtime <- read.table(system.file("retentionIndex/metabolites.txt",
```

Comment:
same here, please change to `system.file("retentionIndex", "metabolites", ...)`.

Reply:
Done.